src/utils.ts

```diff
@@ -179,48 +180,65 @@ export const getStoryIcon = (storyType: string): string => {
  * @returns string
  */
 export const getPrDescription = (body: string = '', story: StoryResponse): string => {
+  if (!shouldPrependPivotalStats(body)) {
```

Comment:
if we do it outside this method then we can prevent the API call itself, right? with this we are triggering a call always

Reply:
and the method is called before it is defined. we should move the method before `getPrDescription`

Reply:
do this check in `main.ts` so that we can avoid calling `updatePrDetails`

Reply:
> if we do it outside this method then we can prevent the API call itself, right? with this we are triggering a call always

which API call? we have to get the PR details to know if it should be added..

Reply:
ohh got it

Reply:
PR details are already part of the triggered action. we aren't making any `GET` call to get the body. 

```js
 if(shouldPrependPivotalStats(prBody)) {
      const prData: PullsUpdateParams = {
        owner: organization.login,
        repo,
        pull_number: prNumber,
        body: getPrDescription(prBody, story),
      };
      updatePrDetails(client, prData);
}
```